doc/tutorial/argument_parser/index.md

```diff
@@ -382,24 +382,51 @@ Add a seqan3::value_list_validator to the `-a/--aggregate-by` option that sets t
 \snippet doc/tutorial/argument_parser/solution6.cpp value_list_validator
 \endsolution
 
-### The seqan3::file_ext_validator
+### The file validator
 
-On construction, the validator receives a list (vector) of valid file extensions.
-The validator throws a seqan3::parser_invalid_argument exception whenever a given filename (string) is not in the given extension list.
+SeqAn3 offers two file validator types: the seqan3::input_file_validator and the seqan3::output_file_validator.
+On construction, the validator receives a list (vector) of valid file extensions that are tested against the extension
+of the parsed option value.
+The validator throws a seqan3::parser_invalid_argument exception whenever a given filename's extension is not in the
+given list of valid extensions. In addition, the seqan3::input_file_validator checks if the file already exists
+or if the specified path does not name a file but a directory. The seqan3::output_file_validator checks if the file does
+not already exists in order to prevent overwriting an already existing file.
```

Comment:
```suggestion
not already exist in order to prevent overwriting an already existing file.
```

In general maybe a little more readable:
```
In addition, the seqan3::input_file_validator checks if the file exists, is a regular file and is readable.
The seqan3::output_file_validator on the other hand ensures that the output does not already exist (in order to prevent overwriting an already existing file) and that it can be created.
```

:nail_care: